<p align="right">
  in conjunction with
  <a href="http://ieeebibm.org/BIBM2025/" target="_blank">IEEE BIBM 2025</a>
</p>


<p align="right">Wuhan, China</p>

<p align="right">December 15-18, 2025</p>

# Description

 **Algorithmic Advances for Single-Cell and Spatial Omics Data Analysis**
With recent technological breakthroughs in single-cell and spatial transcriptomics, as well
as spatial proteomics and other spatially resolved omics platforms, researchers now have
access to molecular data at unprecedented resolution in both cellular identity and spatial
context. These technologies have opened new frontiers in understanding tissue
organization, developmental biology, and disease pathology. However, extracting
meaningful insights from these complex, high-dimensional, and often noisy datasets
requires the development of novel computational methods and algorithmic frameworks.
To date, a wide array of algorithms has been proposed for tasks such as cell type
deconvolution, spatial domain detection, gene-gene interaction modeling, cell-cell
communications, and integrative multi-modal analysis. As spatial and single-cell data
types become increasingly diverse, scalable and robust computational approaches are
needed to bridge modalities, leverage spatial information, and reveal previously
inaccessible layers of biological regulation.
We invite investigators to contribute Original Research articles focused on designing,
applying, or benchmarking algorithms for analyzing single-cell and spatial omics data.
Topics of interest include, but are not limited to, the following:

● Computational methods for analyzing single-cell and spatial transcriptomics data

● Algorithms for spatial domain and tissue architecture identification

● Methods for inferring cell-cell communication and signaling networks in spatial
context

● Methods for exploring the tumor microenvironment from spatial transcriptomics
data

● Joint analysis of spatial transcriptomics, images and spatial proteomics data, or
other omics data

● Machine learning and deep learning approaches for multi-omics integration at
single-cell resolution

● New tools for trajectory inference and lineage tracing with spatial information

● Data-driven frameworks for 3D reconstruction and spatial modeling of tissues

● Denoising, imputation, deconvolution and normalization methods tailored to
spatial and single-cell data

● Algorithmic innovations for cross-sample alignment, batch correction, and
reference mapping

● Applications of advanced computational methods in developmental biology,
neuroscience, immunology, and cancer research

This special issue aims to highlight the state-of-the-art in algorithmic development for
spatial and single-cell omics, and to foster a community of researchers pushing the
boundaries of computational biology through innovative tools and integrative analysis.

Important Dates:

Oct 15, 2025: Due date for workshop paper submission

Nov 10, 2025: Notification of paper acceptance to authors

Nov 23, 2025: Camera-ready of accepted papers

Dec 15-18, 2025: Workshops

# Program Chairs

- **Xin (Maizie) Zhou**, Department of Biomedical Engineering and Computer Science, Vanderbilt University

- **Eric Lu Zhang**, Department of Computer Science, Hong Kong Baptist University

- **Wenji Ma**, Shanghai Jiao Tong University School of Medicine

- **Xiaoqi Zheng**, Shanghai Jiao Tong University School of Medicine

# Program Committee Members

- **Zixuan Cang**, Department of Mathematics, North Carolina State University  

- **Yanxiang Zhao**, Department of Mathematics, George Washington University  

- **Xian Fan Mallory**, Department of Computer Science, Florida State University  

- **Yunfei Hu**, Department of Computer Science, Vanderbilt University  

- **Wenjun Shen**, Department of Bioinformatics, Shantou University  

- **Zhenmiao Zhang**, Department of Computer Science, UCSD  

- **Chao Yang**, Department of Computer Science, Hong Kong Baptist University  

- **Xikang Feng**, School of Software, Northwestern Polytechnic University  

- **Dan Wang**, Department of Data Science, Beijing Normal-Hong Kong Baptist University  

- **Jiaxing Chen**, Department of Computer Science, Beijing Normal-Hong Kong Baptist University  

- **Xiaofei Zhang**, Department of Mathematics, Central China Normal University  

- **Qihuang Zhang**, Department of Epidemiology, Biostatistics and Occupational Health, McGill College  

- **Can Luo**, Department of Biomedical Engineering, Vanderbilt University  

- **Zhenhan Lin**, Department of Computer Science, Vanderbilt University  

# Workshop Submission Requirement

Please submit a full-length paper (up to <font color='red'> 8 page IEEE 2-column format </font>) through the online submission system (you can download the format instruction [here](http://www.ieee.org/conferences_events/conferences/publishing/templates.html).
Electronic submissions (in PDF or Postscript format) are required. Selected participants will be asked to submit their revised papers in a format to be specified at the time of acceptance.  

# Workshop Online Submission:

<!-- **Submit a paper** [Click here](https://wi-lab.com/cyberchair/2021/bibm21/scripts/submit.php?subarea=S09&undisplay_detail=1&wh=/cyberchair/2021/bibm21/scripts/ws_submit.php) -->
**Submit a paper** [Click here](https://wi-lab.com/cyberchair/2025/bibm25/index.php)
