<p align="right">
<table border="1" cellspacing="0" cellpadding="8" width="100%">
  <thead>
    <tr>
      <th style="width:10%">Paper ID</th>
      <th style="width:45%">Title</th>
      <th style="width:45%">Author Names</th>
    </tr>
  </thead>
  <tbody>

    <!-- Moved EMST to FIRST -->
    <tr>
      <td>B1479</td>
      <td>EMST: an interpretable multi-modal model for spatial transcriptomic data analysis</td>
      <td>Rui Han, Weiwei Yuan, Xuan Wang, and Junyi Li</td>
    </tr>

    <tr>
      <td>B328</td>
      <td>scDVCC: deep clustering of scRNA-seq based on dual-view contrastive learning</td>
      <td>Shudong Wang, Yue Song, Wenhao Wu, Hengxiao Li, Yulin Zhang, and Shanchen Pang</td>
    </tr>

    <tr>
      <td>B774</td>
      <td>An Adaptive Single-cell Sequencing data Cluster Method under Weight Fusion Constraint</td>
      <td>Zhenchang Wang, Shasha Yuan, Feng Li, and Juan Wang</td>
    </tr>

    <tr>
      <td>B1857</td>
      <td>scGZDC: Graph-based ZINB Deep Clustering for Single-cell RNA-seq Data</td>
      <td>Hui-Bo Tian, Xiang-Zhen Kong, Jin-Xing Liu, Jun-Liang Shang, Juan Wang, and Ling-Yun Dai</td>
    </tr>

    <tr>
      <td>B1953</td>
      <td>Consistency-Constrained Contrastive Learning with Hard-Negative for Spatial Domain Identification</td>
      <td>Fanghui Zhou, Linjie Wang, Huixia Zhang, and Wei Li</td>
    </tr>

    <tr>
      <td>B2159</td>
      <td>SpaMSC: Multi-Scale Subgraph Contrastive Learning for deciphering Spatial domains in spatial transcriptomics</td>
      <td>Daohui Ge, Haonan Li, Wentian Xin, Feng Li, Ping Wang, and Yuzhuo Yuan</td>
    </tr>

    <tr>
      <td>B2321</td>
      <td>An End-to-End Dual-View Architecture for Spatial Clustering of Spatial Transcriptomics Data by Integrating Histology Images</td>
      <td>Xinru Xu, Shengjun Li, and Juan Wang</td>
    </tr>

  </tbody>
</table>
